### PR TITLE
UXP v6.5 changes

### PR DESCRIPTION
**Pull request by padmkris123**
UXP v6.5 changes

## Description

UXP v6.5 changes

## How Has This Been Tested?

Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/uxp/versions.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ index_desc: General information on UXP and its dependencies
...
-|Premiere Pro|	- | 23.0	| 22.6 | 22.5 || 22.3 | 22.0 |
-|After Effects|	- |	23.0| 22.6 | 22.5 || 22.3 | 22.0 |
-|Animate |	- | -| 23.0 |-|-|- |22.0|
-|Bridge     |	- | 13.0 |-|-|-|-| 12.0|
-|Rush       |	- | |||| ||
-|Aero       |	- | |||| ||
+| Application | UXP v6.5| UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
+| ------------- |  ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+|Photoshop|	24.1 |	24.1 |	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
+|InDesign|	18.1 |	- |	18.0 |	 17.4	|-|-| 17.1 | 17.0|
```
Comment (padmkris123) [inline src/pages/uxp/versions.md:14]: @kareemmyk  Can you verify the row that has Indesign information?

File: src/pages/uxp/versions3P.md [hunk 2]
```diff
@@ -8,17 +8,15 @@ index_desc: General information on UXP and its dependencies for 3rd party plugin
...
-| Application | UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
-| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
-|Photoshop|24.1|	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
-|InDesign|	-|	18.0 |-|-|-|-|-|
-|InDesign Server|	-|	18.0 |-|-|-|-|-|
-|XD|	-|	55 | 54	|-|-|53| 45 |
+| Application | UXP v6.5| UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+|Photoshop|24.1|24.1|	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
+|InDesign|	18.1|	-|	18.0 |	 17.4	|-|-| 17.1 | 17.0|
```
Comment (padmkris123) [inline src/pages/uxp/versions3P.md:14]: @kareemmyk  Can you verify the row that has Indesign information?

File: src/pages/uxp/versions.md (hunk 1 above)
Comment (kareemmyk) [reply src/pages/uxp/versions.md:14]: @padmkris123 .... 
it should be ```-``` for UXP 6.5 as well. Did you receive any confirmation on the version of InDesign?
This is what I have published
| Application | UXP v6.5| UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
|Photoshop|24.1|24.1|	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
|InDesign|	-|	-|	18.0 |	 17.4	|-|-| 17.1 | 17.0|

**Review by pklaschka — APPROVED** [review]
Apart from the versioning stuff, it looks good to me.

I think we have a bit of an issue on the clarity of UXP scripting; "available in scripts" is pretty ambiguous. But I don't think good solutions for this exist, as of right now.

**Pull request merged**